### PR TITLE
Use JLL packages

### PR DESCRIPTION
**Pull request by giordano**
I'd suggest to wait a bit before merging this.  It's quite breaking, now `FFMPEG.ffmpeg` is a function, instead of the path to the FFMPEG executable, which is instead in `FFMPEG.ffmpeg_path`

### CODE REVIEW

**Comment by giordano** [comment]
The failure on 32-bit GNU/Linux is a known problem with JLL packages, we should replicate in the wrappers what we've done in https://github.com/JuliaIO/FFMPEG.jl/commit/53c52cced0130acc3a3639c00625ed98a164719e.  See https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/524

**Comment by asinghvi17** [comment]
Given that this is for Julia 1.3 onwards, do we need `ffmpeg` to be a function (since interpolating it into a command now will also use its environment)?

**Comment by giordano** [comment]
The function `FFMPEG.ffmpeg` is simply re-exported from the JLL package.

File: Project.toml [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
 [deps]
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
-Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+FFMPEG_jll = "b22a6f82-2f65-5046-a5b2-351ab43fb4e5"
 
 [compat]
-julia = "1"
-BinaryProvider = "0.5"
+FFMPEG_jll = "4.1.0"
+julia = "1.3"
```
Comment (asinghvi17) [inline Project.toml]:
```suggestion
julia = "^1.3"
```

**Pull request merged**